lua/gruvbox-material/groups.lua

```diff
@@ -207,6 +207,34 @@ return {
   healthSuccess = { link = "Success" },
   healthWarning = { link = "Warning" },
 
+  DiagnosticError = { link = "Error" },
+  DiagnosticWarn = { link = "Warning" },
+  DiagnosticInfo = { link = "Info" },
+  DiagnosticHint = { link = "Hint" },
+  DiagnosticOk = { link = "Success" },
+  DiagnosticVirtualTextError = { link = "VirtualTextError" },
+  DiagnosticVirtualTextWarn = { link = "VirtualTextInfo" },
+  DiagnosticVirtualTextInfo = { link = "VirtualTextInfo" },
+  DiagnosticVirtualTextHint = { link = "VirtualTextInfo" },
+  DiagnosticVirtualTextOk = { link = "Success" },
+  DiagnosticUnderlineError = { undercurl = true, sp = colors.red },
+  DiagnosticUnderlineWarn = { undercurl = true, sp = colors.yellow },
+  DiagnosticUnderlineInfo = { undercurl = true, sp = colors.blue },
+  DiagnosticUnderlineHint = { undercurl = true, sp = colors.green },
+  DiagnosticUnderlineOk = { undercurl = true, sp = colors.green },
+  DiagnosticFloatingError = { link = "ErrorFloat" },
+  DiagnosticFloatingWarn = { link = "WarningFloat" },
+  DiagnosticFloatingInfo = { link = "InfoFloat" },
+  DiagnosticFloatingHint = { link = "HintFloat" },
+  DiagnosticFloatingOk = { link = "Success" },
```

Comment:
I see a bit of an issue here because floating highlights have a background color set, and Success does not. I would also link HintFloat here, since it already is Green and contains the background color.

Reply:
I think that is the best approach also.

Reply:
In the same vein:
`DiagnosticSignOk = { link = "Success" },`
Should be:
`DiagnosticSignOk = { link = "GreenSign" },`
Right?

Reply:
Ah yes good catch 👍🏽